sqlx-core/src/mssql/transaction.rs

```diff
@@ -14,8 +14,9 @@ pub struct MssqlTransactionManager;
 
 impl TransactionManager for MssqlTransactionManager {
     type Database = Mssql;
+    type Options = ();
```

Comment:
I think I would prefer to see at least an empty `MssqlTransactionOptions` type so that we can safely add things to it later without it being a breaking change.